sklearn/impute.py

```diff
@@ -935,15 +960,13 @@ def transform(self, X):
                 fit_mode=False
             )
             if not (it + 1) % imputations_per_round:
-                if i_rnd >= self.n_burn_in:
-                    Xt += X_filled
                 if self.verbose > 1:
-                    print('[ChainedImputer] Ending imputation round '
+                    print('[IterativeImputer] Ending imputation round '
```

Comment:
This isn't covered in tests, FWIW. It should be covered to ensure there's no AttributeError or whatever

Reply:
Will do.